base/testfiles-ltcmd/ltcmd000.lvt

```diff
@@ -0,0 +1,15 @@
+%
+% Copyright (C) 2017,2021 LaTeX Project
```

Comment:
as I said elsewhere I think copyright lines on .lvt files are kind of overkill and just mean one needs to look after them :-) ... not hurting though, but I would suggest to just drop them.

Reply:
Dilemmas :)  Since these dates are only the dates the files are modified, not `<creation date>-<now>`, it doesn't require manual changing each year/release.  But I for one tend to forget updating these if I change the test file, so I'd favour removing them too.  If nobody objects, I'll remove then